docs/source/changelog/documentation/doc_roi_std.rst

```diff
@@ -0,0 +1,5 @@
+[Documentation] Added ROI and stddev features
+=============================================
+
+ * Added screenshots and description of ROI
```

Comment:
Sorry I didn't notice this in the previous review round: could you also add a reference to the pull request number here? Then it should be ready for merging. Thanks!

Reply:
No problem :+1: